### ISSUE TITLE
fastapi flaskwebgui-keep-alive-server 404 not found

### ISSUE THREAD

**Issue by mrkvn** [ClimenteA/flaskwebgui#83]
![image](https://user-images.githubusercontent.com/52745562/191068187-39e114b0-e5d7-4c88-aef9-66e99073670f.png)

I've run the fastapi [example](https://github.com/ClimenteA/flaskwebgui/tree/master/examples/fastapi-desktop) and I got the error.

The app is alive though. Server is not closing.

I've run both `python main.py` and `python gui.py`. Both the same results.

**Comment by ClimenteA** [comment]
No worries, on the desktop version you won't stay with the console up. 
But, if you want to get rid of that console log error do the following:
- create a GET endpoint `/flaskwebgui-keep-server-alive` (or anything else just make sure to update the js script also with this path);

Please remove from js script the slash: `let route = "/flaskwebgui-keep-server-alive"` to `let route = "flaskwebgui-keep-server-alive"`  (I see it has 2 slashes in the picture).

**Comment by mrkvn** [comment]
Yeah. It's actually no big deal because it's a console error. Just thought you might want to know.

I've already tried this `let route = "flaskwebgui-keep-server-alive"` but didn't work, though I've already managed to remove the error altogether. I've used `let route = ""` and instead of returning `response.json()`, I just return `response`. Because after I've managed to remove the error, I got a json error. Or might want to just return nothing at all to remove the error
